_schema.graphql

```diff
@@ -3173,7 +3253,7 @@ type FilterArtworks implements Node {
     last: Int
   ): ArtworkConnection
     @deprecated(
-      reason: "Favour artwork connections that take filter arguments."
+      reason: "Use only for filtering over ElasticSearch-backed fields, otherwise favor artwork connections that take filter arguments."
```

Comment:
Yup totally makes sense to use ES in this case

Reply:
Not sure, but I think this may conflict with or overwrite [this very recent change](https://github.com/artsy/metaphysics/pull/1418/files#diff-922d7e3f4c475706810cc27a8bdddba2) due to a tooling issue — would you mind double-checking?

Reply:
Nice catch - I noticed this while rebasing but resolved it incorrectly. Should be fixed and commented out again.